examples/armada-manifest-v1.yaml

```diff
@@ -0,0 +1,101 @@
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-dependecy
+data:
+  name: helm-toolkit
+  release: helm-toolkit
+  namespace: null
+  values: {}
+  source:
+    type: git
+    location: git://github.com/openstack/openstack-helm
+    subpath: helm-toolkit
+    reference: master
+  dependecies: []
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-mariadb
+data:
+  name: mariadb
+  release: mariadb
+  namespace: undercloud
+  timeout: 3600
+  values: {}
+  source:
+    type: git
+    location: git://github.com/openstack/openstack-helm
+    subpath: mariadb
+    reference: master
+  dependecies:
+    - ucp-dependecy
+---
+schema: armada/Chart/v1
+metadata:
+  schema: metadata/Document/v1
+  name: ucp-memcache
+data:
+  name: memcache
```

Comment:
I do think that utilizing the metadata info, would simplify things.
chart versioning is managed by the reference field.